source/compiler/qsc_frontend/src/resolve/tests.rs

```diff
@@ -65,6 +66,16 @@ impl<'a> Renamer<'a> {
     fn rename(&self, input: &mut String) {
         for (span, change) in self.changes.iter().rev() {
             let name = match change {
+                Change::Import(res_source, res) => {
+                    format!(
+                        "{} as {}",
```

Comment:
nitpick: I was a little confused by this phrasing at first. I think I understand now that it's meant to convey "<item id of the target of the export> as <item id of the export itself>" but at first I thought it was related to aliases. Is there a different phrasing that could make it less overlapping with the existin alias syntax?

Reply:
Given this context, I now love this change. They look like aliases because they are aliases.